src/scripts/main.js

```diff
@@ -1,17 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.body;
+
+  body.insertAdjacentHTML('afterbegin', `
+    <div
+      class="notification ${type}"
+      style="position: absolute; top: ${posTop}; right: ${posRight}">
+      <h2 class="title">${title}</h2>
+      <p>${description}</p>
+    </div>
+  `);
+
+  const notification = document.querySelector('.notification');
+
+  setTimeout(() => notification.add(), 2000);
 };
 
-pushNotification(10, 10, 'Title of Success message',
+pushNotification('10px', '10px', 'Title of Success message',
```

Comment:
do not modify these arguments

Reply:
If I don`t modify these argument code doesn`t work!